database/clickhouse/clickhouse.go

```diff
@@ -193,8 +198,12 @@ func (ch *ClickHouse) SetVersion(version int, dirty bool) error {
 		return err
 	}
 
-	query := "INSERT INTO " + ch.config.MigrationsTable + " (version, dirty, sequence) VALUES (?, ?, ?)"
-	if _, err := tx.Exec(query, version, bool(dirty), time.Now().UnixNano()); err != nil {
+	query := "INSERT INTO " + ch.config.MigrationsTable + " (version, dirty, sequence) VALUES"
+	batch, err := tx.Prepare(query)
```

Comment:
Why does prepare need to be used?

Reply:
Since V2 supports insert request only in "prepared statements" or in asynchronous mode

Reply:
Using numeric placeholders or named placeholder is also supported. Just FI.